Makefile

```diff
@@ -93,7 +93,10 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
-run: manifests generate fmt vet ## Run a controller from your host.
+install-crds: ## Install CRDs
+	-kubectl create -f config/crd/bases/
```

Comment:
`-` at the start of this line ignores any errors on this command.

This is added to ignore the errors like "CRDs already installed"

Reply:
And what about legitimate errors? Such as, cannot connect to cluster or permission denied? What if the CRDs were already installed, but required updates?

I would suggest to use `apply` instead of `create`, and to not ignore errors.

Reply:
ah, so stupid of me. Thanks @jannfis

Reply:
Oh, and there is already `install` target that does install the CRDs into the cluster. Have you considered using that?

Reply:
Thanks for that, I did not realise. So, either we can include the `install` as part of `make run`(updated the PR) or simply close the PR and users would run `make install` and then `make run` ?
Please feel free to close or approve the PR :)

Reply:
I would not alter the behavior of `make run`.   It is probably a good idea not re-apply the manifests in every run and let the install be explicit.   Let's close it.